ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -371,12 +376,68 @@ def add_capacity(osd_size_capacity_requested):
     storageDeviceSets->count = (capacity reqested / osd capacity ) + existing count storageDeviceSets
 
     """
-
+    osd_size_existing = get_osd_size()
+    device_sets_required = int(osd_size_capacity_requested / osd_size_existing)
+    lvpresent = check_local_volume()
+    if lvpresent:
+        ocp_obj = OCP()
+        workers = get_typed_nodes(node_type='worker')
+        worker_names = [worker.name for worker in workers]
+        output = ocp_obj.exec_oc_cmd("get localvolume local-block -n local-storage -o yaml")
+        cur_device_list = output["spec"]["storageClassDevices"][0]['devicePaths']
```

Comment:
Small nitpick here but let's be consistent with the use of `"` or `'`. You are using both for different keys in this line, let's stick to one or the other. Check for other instances of this as well.

Reply:
Fixed it!